app/assets/stylesheets/search.scss

```diff
@@ -126,7 +138,7 @@
   border-radius: 4px;
   padding:  1em;
   position: relative;
-  width: 90% !important;
+  width: 80% !important;
```

Comment:
Properties should be ordered background-color, border-radius, padding, position, top, width